ando/tools/generator/tests/test_AnDOGenerator.py

```diff
@@ -120,11 +120,17 @@ def test_metadata_files(self):
         self.ando_data.generate_structure()
         self.ando_data.register_metadata_files(*self.test_mdata_files)
         self.ando_data.generate_metadata_files()
+        self.ando_data.create_metadata_file_participants()
```

Comment:
I think the `create_metadata_file_xxx` methods should be collectively called by another method to not make life more complicated than necessary. See also my comment about renaming `generate_metadata_files`...

Reply:
done